tests/unit/test_tasks.py

```diff
@@ -32,14 +32,14 @@ def test_task_pegatweets(self):
         self.assertEqual(res, 200)
         self.assertGreater(len(resp), 0)
 
-    def test_task_fetch_redemet(self):
```

Comment:
**test_task_fetch_redemet()** Is a valid name because there is more than one weather variable to capture...